Source/cmake/OptionsWin.cmake

```diff
@@ -106,6 +103,7 @@ WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_XSLT PUBLIC ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_SMOOTH_SCROLLING PRIVATE OFF)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEBGL PRIVATE OFF)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(USE_AVIF PRIVATE OFF)
+WEBKIT_OPTION_DEFAULT_PORT_VALUE(USE_SKIA PRIVATE ON)
```

Comment:
Only the default should change no other changes to OptionsWin.cmake are required.

Reply:
It doesn't work because `find_package` is placed before this line. But, I wont't fix the problem in this time because I have no plan to maintain Windows port cairo code anymore after this change.
This is not blocked by https://github.com/WebKit/WebKit/pull/36366 .

Reply:
I am working with your managers to resolve this issue. At this time this is blocked.